osu.Game.Tournament/Models/TournamentPlayer.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Game.Database;
+using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Users;
+
+namespace osu.Game.Tournament.Models
+{
+    /// <summary>
+    /// A tournament player, containing simple information about the player.
+    /// </summary>
+    [Serializable]
+    public class TournamentPlayer : IUser
+    {
+        public int Id { get; set; }
+
+        public string Username { get; set; } = string.Empty;
+
+        /// <summary>
+        /// The player's country.
+        /// </summary>
+        public Country? Country { get; set; }
+
+        /// <summary>
+        /// The player's global rank, or null if not available.
+        /// </summary>
+        public int? Rank { get; set; }
+
+        /// <summary>
+        /// A URL to the player's profile cover.
+        /// </summary>
+        public string CoverUrl { get; set; } = string.Empty;
+
+        public APIUser ToUser()
+        {
+            var user = new APIUser
+            {
+                Id = Id,
+                Username = Username,
+                Country = Country,
+                CoverUrl = CoverUrl,
+            };
+
+            user.Statistics = new UserStatistics
+            {
+                User = user,
+                GlobalRank = Rank
+            };
+
+            return user;
+        }
```

Comment:
I don't really like the addition of such method, but it's required to provide a user panel display, which can only accept `APIUser`. Might be improvable using an interface, or by fetching user during construction of panel. But went with this approach to avoid complexity.

Reply:
I think it's fine for now.